Resources/Prototypes/Entities/Objects/Materials/bluespace.yml

```diff
@@ -54,3 +54,60 @@
   components:
   - type: Stack
     count: 5
+
+- type: entity
+  parent: MaterialBase
+  id: MaterialNormality
+  suffix: Full
+  name: normality crystal
+  components:
+  - type: Sprite
+    sprite: Nyanotrasen/Objects/Materials/materials.rsi
```

Comment:
Meh im using Nyanotrasen codespace but sure.

Reply:
eh, ill move it, was just feeling lazy and VM had left it like that